src/Countries/US/NewMexico/NewMexicoWorkersCompensation/V20190101/NewMexicoWorkersCompensation.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\NewMexico\NewMexicoWorkersCompensation\V20190101;
+
+use Appleton\Taxes\Countries\US\NewMexico\NewMexicoWorkersCompensation\NewMexicoWorkersCompensation as BaseNewMexicoWorkersCompensation;
+use Illuminate\Database\Eloquent\Collection;
+
+class NewMexicoWorkersCompensation extends BaseNewMexicoWorkersCompensation
+{
+    public function compute(Collection $tax_areas)
+    {
+        if ($this->payroll->getStartDate()->weekOfMonth === $this->payroll->getStartDate()->endOfQuarter()->weekOfMonth) {
+            return 2.30;
```

Comment:
Thought it was weird that the same amount was taken out for employee and employer so I looked and the AC says 2.00 for employee and 2.3 for employer.

Reply:
Thanks, good catch.

Reply:
should these be calling withholdTax as well?

Reply:
@stevespur Yes. We need to make sure to call that for any taxes that are withheld (deductions). Good catch.